<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>678</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>678</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>671</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_end">
       <property name="text">
        <string>End REC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_start">
       <property name="text">
        <string>Start REC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_process">
       <property name="text">
        <string>Process</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_camera">
       <property name="text">
        <string>Open Camera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_open">
       <property name="text">
        <string>Open File</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1091</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>973</x>
      <y>550</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>510</y>
      <width>256</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuopen">
    <property name="title">
     <string>open</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuedit">
    <property name="title">
     <string>edit</string>
    </property>
   </widget>
   <addaction name="menuopen"/>
   <addaction name="menuedit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_open</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_pushButton_open_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>937</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>797</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_camera</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_pushButton_camera_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>938</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>798</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_process</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_pushButton_process_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>940</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_pushButton_start_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>945</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>801</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_end</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_pushButton_end_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>939</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>802</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_pushButton_open_clicked()</slot>
  <slot>on_pushButton_camera_clicked()</slot>
  <slot>on_pushButton_process_clicked()</slot>
  <slot>on_pushButton_start_clicked()</slot>
  <slot>on_pushButton_end_clicked()</slot>
 </slots>
</ui>
